ontology/yaml/resources/HVAC/entity_types/GENERALTYPES.yaml

```diff
@@ -51,6 +51,9 @@ FCU:
   - failed_communication_alarm
   - master_alarm
   - filter_alarm
+  - lost_power_alarm
```

Comment:
TBD on this - is there a better way to capture general alarms/do we even care about all of these alarms in the first place? Review w Trevor and on-site team